src/App.jsx

```diff
@@ -26,20 +27,15 @@ export const alex = {
 export const App = () => (
   <div className="App">
     <section className="Person">
```

Comment:
you already have section inside `Person` component, remove it and leave just  3 components